.gitignore

```diff
@@ -1,6 +1,7 @@
 .idea
 bin
 obj
+src/App/node_modules/
```

Comment:
We don't have such a folder in the repo.

Reply:
Was a leftover from old app in my folder. Reverted